pyproject.toml

```diff
@@ -18,9 +18,12 @@ rq = "^1.15.1"
 pyyaml = "^6.0.1"
 redis = "^4.6.0"
 
+[tool.poetry.group.docs]
+optional = true
```

Comment:
Ah nice

Check the poetry install on the devcontainer file, and tell it to install all groups of dependencies.

Reply:
ok

Reply:
`poetry install --no-interaction --no-ansi --no-root`
it is already installing the dependencies